jasmin_services/views.py

```diff
@@ -442,6 +443,18 @@ def role_apply(request, service, role, bool_grant = None, previous=None):
                 "You have already have an active request for the specified grant"
             )
             return redirect_to_service(service)
+
+    # ONLY FOR CEDA SERVICES: has an active request for this chain it must be rejected
+    licence_url = None
+    if request.user._wrapped.__class__.__name__ == 'CedaUser':
```

Comment:
Don't think this is the best way to check if we're on the CEDA or JASMIN side but couldn't think of another way. Other than a flag in the settings?